src/components/taskDetails/Details.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC } from 'react';
+import classNames from './task-details.module.scss';
+
+function setDetailsColor(value: string = '') {
+  switch (value) {
+    case 'HIGH':
+      return 'red';
+    case 'MEDIUM':
+      return 'orange';
+    case 'LOW':
+      return 'green';
+    default:
+      return 'black';
+  }
+}
+
+type Props = {
+  detailType: string;
+  value?: string;
+};
+
+const Details: FC<Props> = ({ detailType, value = '' }) => {
+  return (
+    <div>
+      <span className={classNames['block_content_detail_type']}>
```

Comment:
i think classname here can be `section_content` and maybe you can wrap inside section element and add some `ARIA` attribute to make it meaningful as empty `section` without any attribute is same as `div`

Reply:
done